DM.MovieApi/MovieDb/Movies/ApiMovieRequest.cs

```diff
@@ -152,7 +152,51 @@ public async Task<ApiQueryResponse<MovieCredit>> GetCreditsAsync( int movieId, s
 
             string command = $"movie/{movieId}/credits";
 
-            ApiQueryResponse<MovieCredit> response = await base.QueryAsync<MovieCredit>( command, param );
+            ApiQueryResponse<MovieCredit> response = await base.QueryAsync<MovieCredit>(command, param);
+
+            return response;
+        }
+
+        public async Task<ApiSearchResponse<MovieInfo>> GetRecommendationsAsync(int movieId, int pageNumber = 1, string language = "en")
+        {
+            var param = new Dictionary<string, string>
+            {
+                {"language", language},
+                {"append_to_response", "keywords"},
+
+            };
+
+            string command = $"movie/{movieId}/recommendations";
+            ApiSearchResponse<MovieInfo> response = await base.SearchAsync<MovieInfo>(command, pageNumber, param);
+
+            if (response.Error != null)
+            {
+                return response;
+            }
+
+            response.Results.PopulateGenres(_genreApi);
+
+            return response;
+        }
+
+        public async Task<ApiSearchResponse<MovieInfo>> GetSimilarAsync(int movieId, int pageNumber = 1, string language = "en")
+        {
+            var param = new Dictionary<string, string>
+            {
+                {"language", language},
+                {"append_to_response", "keywords"},
```

Comment:
If using "append_to_response", then the backing integration tests must test the "keywords" are present in the response.